src/main.cpp

```diff
@@ -2049,11 +2049,12 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         // Note the std::max is there to deal with the rollover of BlockV12Height + DISCONNECT_GRACE_PERIOD if
         // BlockV12Height is set to std::numeric_limits<int>::max() which is the case during testing.
         if (pfrom->nVersion < MIN_PEER_PROTO_VERSION
-                || (DISCONNECT_OLD_VERSION_AFTER_GRACE_PERIOD
-                    && pfrom->nVersion < PROTOCOL_VERSION
-                    && pindexBest->nHeight > std::max(Params().GetConsensus().BlockV12Height,
-                                                      Params().GetConsensus().BlockV12Height + DISCONNECT_GRACE_PERIOD)))
```

Comment:
This change is formatting only, correct?

Reply:
Yep.

Reply:
I originally put the extra checks on sub version here, but released it hadn't been serialized yet.